ui/src/pages/AllStories/index.tsx

```diff
@@ -99,16 +145,44 @@ function createData({
     image_url,
     video_url,
     content,
+    tags,
   };
 }
 
 export const AllStories: React.FC = () => {
+  const [anchorEl, setAnchorEl] = React.useState<HTMLButtonElement | null>(null);
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const open = Boolean(anchorEl);
+  const id = open ? 'simple-popover' : undefined;
```

Comment:
`undefined` ? Should it just be `null` or `""`?

Reply:
dont think it would matter. I just copied it from https://material-ui.com/components/popover/